www/include/options/session/connected_user.php

```diff
@@ -58,40 +132,71 @@
 $tpl = initSmartyTpl($path, $tpl);
 
 $session_data = array();
-$query = "SELECT session.*, contact_name, contact_admin FROM session, contact " .
-    "WHERE contact_id = user_id ORDER BY contact_name, contact_admin";
-$res = $pearDB->query($query);
-for ($cpt = 0; $r = $res->fetchRow(); $cpt++) {
+$res = $pearDB->query(
+    "SELECT session.*, contact_name, contact_admin, contact_auth_type, `ldap_last_sync`
+    FROM session, contact
+    WHERE contact_id = user_id ORDER BY contact_name, contact_admin"
+);
+for ($cpt = 0; $r = $res->fetch(); $cpt++) {
     $session_data[$cpt] = array();
     if ($cpt % 2) {
         $session_data[$cpt]["class"] = "list_one";
     } else {
         $session_data[$cpt]["class"] = "list_two";
     }
-
     $session_data[$cpt]["user_id"] = $r["user_id"];
     $session_data[$cpt]["user_alias"] = $r["contact_name"];
     $session_data[$cpt]["admin"] = $r["contact_admin"];
     $session_data[$cpt]["ip_address"] = $r["ip_address"];
-    $session_data[$cpt]["last_reload"] = date("H:i:s", $r["last_reload"]);
+    $session_data[$cpt]["last_reload"] = $r["last_reload"];
+    $session_data[$cpt]["ldapContact"] = $r['contact_auth_type'];
 
-    $query = "SELECT topology_name, topology_page, topology_url_opt FROM topology WHERE topology_page = '" .
-        $r["current_page"] . "'";
-    $resCP = $pearDB->query($query);
-    $rCP = $resCP->fetchRow();
+    $resCP = $pearDB->prepare(
```

Comment:
I think you can place this subquery directly in the first request to relieve the database (SELECT session...)

Reply:
if current_page = 0  (eg: the user have acl misconfigurations), the request will be inconsistent and the contact won't be displayed properly (or at all)